### DISCUSSION TITLE
Not publishing to nuget.org?

### DISCUSSION THREAD

**Discussion by jamescrosswell** [mattleibow/DeviceRunners#35] [General]
I see the device runners are currently published as a NuGet package to the Azure Artifacts service as a custom feed.
https://github.com/mattleibow/DeviceRunners/blob/e14622129f2618167da92484cfbd6f0ac83da628/.github/workflows/pack.yml#L45-L49

This makes it somewhat more difficult to consume. 

Out of curiosity, is there any reason not to publish instead/also to NuGet.org?

**Comment by mattleibow** [comment]
No reason. I had an idea to move this into the community toolkits repo, but that may have been to big a dream. Until this repo gets a lot of traction or there is a need to move this into the wider space, it probably will live here.

I do need to verify that I like my namespaces and nuget IDs now that it is not going to change.

I will have a look at this and see if I can get it to nuget soon.
